sale_product_pack/README.rst

```diff
@@ -78,6 +78,7 @@ Authors
 * NaN·tic
 * ADHOC SA
 * Tecnativa
+* bloopark systems GmbH & Co. KG
```

Comment:
In my opinion, it is ok if you add yourself as a contributor but the authors are which ones created the module or made stronger modifications (refactors)